test/colab/fix-requirements.sh

```diff
@@ -0,0 +1,19 @@
+cp test/colab/requirements.txt build/colab-test/requirements-fixed.txt
+
+# Ensure setuptools version lower than 58 is used, as 58 and above break the install due to use_2to3 not being available
+sed -i'' -e "1s/^/setuptools<58\n/" build/colab-test/requirements-fixed.txt
+
+# Remove Tensorflow libraries due to AVX missing on M1 mac
+sed -i'' -e "s/kapre==/# kapre==/" build/colab-test/requirements-fixed.txt
+sed -i'' -e "s/keras==/# keras==/" build/colab-test/requirements-fixed.txt
+sed -i'' -e "s/tensorflow==/# tensorflow==/" build/colab-test/requirements-fixed.txt
+
+# Remove non-existing versions
+sed -i'' -e "s/google-colab @/# google-colab @/" build/colab-test/requirements-fixed.txt
```

Comment:
Instead of removing, should we try installing from [here](https://github.com/googlecolab/colabtools) ?

Reply:
I can look into that, are you thinking of always pulling from master here, or manually pinning to the latest commit when getting the requirements from colab?

Reply:
I guess `master` should be good as this is probably what they use when they build their images.